packages/toolpad-studio/src/toolpad/AppEditor/PageEditor/RenderPanel/RenderOverlay.tsx

```diff
@@ -1160,6 +1160,17 @@ export default function RenderOverlay({ bridge }: RenderOverlayProps) {
     };
   }, [handleNodeDragEnd]);
 
+  const scrollSelectedNode = React.useCallback(() => {
+    if (!selectedNode) {
+      return;
+    }
+    bridge?.canvasCommands.scrollComponent(selectedNode?.id as string);
```

Comment:
```suggestion
    bridge?.canvasCommands.scrollComponent(selectedNode.id as string);
```

I think this `?` shouldn't be needed as you already checked above if it's truthy.